src/components/common-components/ProjectBlock/ProjectBlock.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+import './ProjectBlock.css';
+import ProjectModal from '../ProjectModal/ProjectModal'
+
+/**
+ * @returns {React.Component}
+ */
+const ProjectBlock = props => {
```

Comment:
Some of the code in this class looks similar to the code above-- can they be refactored to be reused somehow?